### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by ennamarie19**
Initial PR

### CODE REVIEW

File: kramer.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def __init__(self:object,_n1_:{choice(_types_)}=False,_n2_:{choice(_types_)}=0,*
```
Comment (xansec) [inline kramer.py:169]: Can we optionally comment this out (I assume this is for fuzzing)?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: kramer.py (hunk 1 above)
Comment (capuanob) [reply kramer.py:169]: Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**